.pre-commit-config.yaml

```diff
@@ -0,0 +1,47 @@
+fail_fast: false
+default_language_version:
+    python: python3
+default_stages:
+-   commit
+-   push
+minimum_pre_commit_version: 2.10.1
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+    -   id: black
+        additional_dependencies: [toml]
+-   repo: https://github.com/timothycrosley/isort
+    rev: 5.7.0
+    hooks:
+    -   id: isort
+        additional_dependencies: [toml]
+        args: [--order-by-type]
+-   repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+    -   id: autoflake
+        exclude: .*notebooks.*
+        args: [--in-place, --remove-all-unused-imports, --remove-unused-variable, --ignore-init-module-imports]
+-   repo: https://gitlab.com/pycqa/flake8
```

Comment:
Is this hook necessary? Especially after transferring the into to the setup.cfg?